src/metabase/api/collection.clj

```diff
@@ -361,15 +361,18 @@
 
 (api/defendpoint POST "/"
   "Create a new Collection."
-  [:as {{:keys [name color description parent_id namespace]} :body}]
+  [:as {{:keys [name color description parent_id namespace type]} :body}]
```

Comment:
Another reason to use something other than `type` is so you wouldn't end up overshadowing `clojure.core/type` everywhere... either Eastwood or Bikeshed complains about that in regular `defn` forms. I guess it can't catch it here tho